distribution/integration/tests-new/pom.xml

```diff
@@ -20,7 +20,7 @@
                 <inherited>false</inherited>
                 <configuration>
                     <!--<argLine>-Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m -Xdebug -Xnoagent -Djava.compiler=NONE -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=8000</argLine>-->
-                    <argLine>-Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m</argLine>
+                    <!--<argLine>-Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m</argLine>-->
```

Comment:
Yea, so can you only remove the MaxPermSize and PermSize? We should keep the Xmx attribute.